CHANGELOG.unreleased.md

```diff
@@ -15,6 +15,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Host/Network] Fix 'an error as occured' briefly displaying in 'Mode' column of the host's Network tab (PR [#7954](https://github.com/vatesfr/xen-orchestra/pull/7954))
```

Comment:
This should be 'an error has occurred'

Reply:
Also, use backticks `` ` `` to indicate verbatim text.